pkg/assembler/backends/ent/backend/hashequal.go

```diff
@@ -61,6 +62,18 @@ func (b *EntBackend) IngestHashEqual(ctx context.Context, artifact model.Artifac
 	return toModelHashEqual(record.Unwrap()), nil
 }
 
+func (b *EntBackend) IngestHashEquals(ctx context.Context, artifacts []*model.ArtifactInputSpec, otherArtifacts []*model.ArtifactInputSpec, hashEquals []*model.HashEqualInputSpec) ([]*model.HashEqual, error) {
+	var result []*model.HashEqual
+	for i := range hashEquals {
+		he, err := b.IngestHashEqual(ctx, *artifacts[i], *otherArtifacts[i], *hashEquals[i])
+		if err != nil {
+			return nil, gqlerror.Errorf("IngestHashEquals failed with err: %v", err)
```

Comment:
The error returned from the function does not provide much context on which call failed. 

Could you please consider returning the index that failed?

Reply:
Sure, good idea, thanks.